docs/content/how-to/agent/create-agent-cluster.md

```diff
@@ -398,9 +608,142 @@ spec:
+clusteroperator.config.openshift.io/csi-snapshot-controller                    4.11.5    True        False         False      25m     
+clusteroperator.config.openshift.io/dns                                        4.11.5    True        False         False      23m     
+clusteroperator.config.openshift.io/image-registry                             4.11.5    True        False         False      23m     
+clusteroperator.config.openshift.io/ingress                                    4.11.5    True        False         False      53m     
+clusteroperator.config.openshift.io/insights                                   4.11.5    True        False         False      25m     
+clusteroperator.config.openshift.io/kube-apiserver                             4.11.5    True        False         False      54m     
+clusteroperator.config.openshift.io/kube-controller-manager                    4.11.5    True        False         False      54m     
+clusteroperator.config.openshift.io/kube-scheduler                             4.11.5    True        False         False      54m     
+clusteroperator.config.openshift.io/kube-storage-version-migrator              4.11.5    True        False         False      25m     
+clusteroperator.config.openshift.io/monitoring                                 4.11.5    True        False         False      21m     
+clusteroperator.config.openshift.io/network                                    4.11.5    True        False         False      25m     
+clusteroperator.config.openshift.io/openshift-apiserver                        4.11.5    True        False         False      54m     
+clusteroperator.config.openshift.io/openshift-controller-manager               4.11.5    True        False         False      54m     
+clusteroperator.config.openshift.io/openshift-samples                          4.11.5    True        False         False      23m     
+clusteroperator.config.openshift.io/operator-lifecycle-manager                 4.11.5    True        False         False      54m     
+clusteroperator.config.openshift.io/operator-lifecycle-manager-catalog         4.11.5    True        False         False      54m     
+clusteroperator.config.openshift.io/operator-lifecycle-manager-packageserver   4.11.5    True        False         False      54m     
+clusteroperator.config.openshift.io/service-ca                                 4.11.5    True        False         False      25m     
+clusteroperator.config.openshift.io/storage                                    4.11.5    True        False         False      25m     
+~~~
+
+## Enabling Node Auto-Scaling for the Hosted Cluster
+
+Auto-scaling can be enabled, if we enable auto-scaling when more capacity is required in our Hosted Cluster a new Agent will be installed (providing that we have spare agents). In order to enable auto-scaling we can run the following command:
```

Comment:
If we choose to enable auto-scaling, when more capacity is required...

Reply:
Fixed.